documentation/tut/forecasting_scheduling.rst

```diff
@@ -24,16 +24,16 @@ Start to run one worker for each kind of job (in a separate terminal):
 
 .. code-block:: console
 
-   flexmeasures run-worker --queue forecasting
-   flexmeasures run-worker --queue scheduling
+   flexmeasures jobs run-worker --queue forecasting
+   flexmeasures jobs run-worker --queue scheduling
 
 
 You can also clear the job queues:
 
 .. code-block:: console
 
-   flexmeasures clear-queue --queue forecasting
-   flexmeasures clear-queue --queue scheduling
+   flexmeasures jobs clear-queue --queue forecasting
+   flexmeasures jobs clear-queue --queue scheduling
```

Comment:
Would be nice of we could also clear the `failed` queue with this command. I currently get:
`ValueError: Unknown queue 'failed'.`